src/wallet/wallet.h

```diff
@@ -136,6 +136,8 @@ static const bool DEFAULT_DISABLE_WALLET = false;
 static const bool DEFAULT_WALLETCROSSCHAIN = false;
 //! -maxtxfee default
 constexpr CAmount DEFAULT_TRANSACTION_MAXFEE{COIN / 10};
+//! maxfeerate default
+const CFeeRate DEFAULT_TRANSACTION_MAXFEERATE(CFeeRate((COIN / 10), 1000));
```

Comment:
1000 is redundant

Reply:
Yes, fixed here and other place I did the same.